jerry-debugger/src/lib/protocol-handler.ts

```diff
@@ -119,6 +121,10 @@ export class JerryDebugProtocolHandler {
     this.lastBreakpointExact;
   }
 
+  getSource() {
+    return this.source;
```

Comment:
Is it possible `this.source` is not yet valid when the controller calls this function?

For readability, could you add the return type to the function? I'm guessing it's `: string`?